migrations/versions/215392b11b0e_.py

```diff
@@ -0,0 +1,44 @@
+"""empty message
+
+Revision ID: 215392b11b0e
+Revises: 824268a7a675
+Create Date: 2019-03-20 17:11:43.674304
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '215392b11b0e'
+down_revision = '824268a7a675'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_index('idx_project_info composite', 'project_info', ['locale', 'project_id'], unique=False)
+    op.create_index('textsearch_idx', 'project_info', ['text_searchable'], unique=False)
+    op.add_column('projects', sa.Column('map_editors', postgresql.ARRAY(sa.Integer()), nullable=True))
+    op.add_column('projects', sa.Column('validate_editors', postgresql.ARRAY(sa.Integer()), nullable=True))
+    op.alter_column('projects', 'task_creation_mode',
+               existing_type=sa.INTEGER(),
+               nullable=False)
+    op.create_index(op.f('ix_projects_map_editors'), 'projects', ['map_editors'], unique=False)
+    op.create_index(op.f('ix_projects_validate_editors'), 'projects', ['validate_editors'], unique=False)
```

Comment:
do we need indices on these columns? For use cases where you want to search for all projects with "iD X" enabled, does this index support that?

Reply:
Yes. I put that in so projects would be searchable based on editors in the future. I tried to mimic similar implementations.

Reply:
👍 sounds good!